### PR TITLE
Support csap kalkaal consent

### PR DESCRIPTION
**Pull request by IsaackMwenda**
We sent a consent message to CSAP and KALKAAL contacts. We will export consented individuals for this week's sms advert.

### CODE REVIEW

File: code_schemes/csap_kalkaal_consent.json [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        "CodeID": "code-S-461bfcdf",
+        "CodeType": "Meta",
+        "MetaCode": "statement",
+        "DisplayText": "meta: statement",
+        "NumericValue": -100090,
+        "StringValue": "statement",
+        "VisibleInCoda": true,
+        "Shortcut": "s"
+      },
+      {
```
Comment (as2388) [inline code_schemes/csap_kalkaal_consent.json:36]: How are you labelling people who respond yes. Opt-in?

**Review by as2388 — APPROVED** [review]
Next time we do this we may want to think about the effect of responding immediately/labelling all responses then responding. Between this and CSAP we'll have tried it both ways.

File: code_schemes/csap_kalkaal_consent.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/csap_kalkaal_consent.json:36]: yes Meta: Optin

**Comment by IsaackMwenda** [comment]
> Next time we do this we may want to think about the effect of responding immediately/labelling all responses then responding. Between this and CSAP we'll have tried it both ways.

We choose this option because making the decision of who opted in or not in Rapidpro using regexes meant we might lose participants who don't explicitly answer YES and vice-versa.

**Pull request merged**